docs/src/routes/docs/components/breadcrumbs.svx

```diff
@@ -0,0 +1,101 @@
+---
+name: Breadcrumbs
+---
+
+<script>
+  import { Breadcrumbs, Button, Chip } from 'attractions';
+  import { ArrowRightIcon, HomeIcon } from 'svelte-feather-icons';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+
+  const items = [
+    { href: '/', text: 'Home' },
+    { href: '/docs', text: 'Docs' },
+    { text: 'Components' },
+  ];
+
+  const itemsType = 'Array<{ href?: string; text: string; }>';
+</script>
+
+# Breadcrumbs
+
+Breadcrumbs are used to indicate the current page's location in the navigation hierarchy, as well as to navigate through it.
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <Breadcrumbs {items} />
+    <Breadcrumbs {items} separator='~' />
+    <Breadcrumbs {items}>
+      <div slot="item" let:item>
+        {#if item.href === '/'}
+          <Button href="/" round small>
+            <HomeIcon size="20" />
+          </Button>
+        {:else}
+          <Chip>{item.text}</Chip>
+        {/if}
+        </div>
+      <ArrowRightIcon slot="separator" size="20" />
+    </Breadcrumbs>
+  </div>
+  <div slot="source">
+
+```svelte
+<script>
+  const items = [
+    { href: '/', text: 'Home' },
+    { href: '/docs', text: 'Docs' },
+    { text: 'Components' },
+  ];
+</script>
+
+<Breadcrumbs {items} />
+
+<Breadcrumbs {items} separator='~' />
+
+<Breadcrumbs {items}>
+  <div slot="item" let:item>
+    {#if item.href === '/'}
+      <Button href="/" round small>
+        <HomeIcon size="20" />
+      </Button>
+    {:else}
+      <Chip>{item.text}</Chip>
+    {/if}
+  </div>
+  <ArrowRightIcon slot="separator" size="20" />
+</Breadcrumbs>
+```
+
+  </div>
+</Showcase>
+
+
+## Properties {#properties}
+
+### Functional Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`items`** | `[]` | <code>{itemsType}</code> | The list of segments to be displayed. |
+| **`separator`** | `'/'` | `string` | The separator between segments. |
+
+
+## Slots {#slots}
+
+### `item` slot
+
+The item displayed in the breadcrumbs. Defaults to a `<Button>` if `href` is provided, otherwise a `<span>`.
+
+  | Prop Name | Type | Description |
+  | --------- | ---- | ----------- |
+  | **`item`** | `{ href?: string, text: string }` | Information about the current segment. |
+
+### `separator` slot
+
+If a string separator is not enough, use this slot to add a custom component separator (and ignore the prop).
+
+## SCSS Variables {#scss-variables}
+
+| Name | Description | Default |
+| ---- | ----------- | ------- |
+| **`$thin-font-weight`** | Description. | `300` |
```

Comment:
```suggestion
| **`$thin-font-weight`** | The font size of the separator and the segments without `href`. | `300` |
```

Reply:
Whoops. That's what happens when you just copy the template :D